CHANGELOG.md

```diff
@@ -26,13 +27,50 @@ This release contains the definition and documentation of
 * Application Profiles API v0.3.0
 
 The API definition(s) are based on
-* Commonalities v0.4.0-rc.2
-* Identity and Consent Management v0.2.0-rc.2
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.0
 
-## Connectivity Insights API v0.4.0-rc.1
+*  **r1.2 release of connectivity insights has the following API definitions:**
 
-0.4.0-rc.1 is the first release of the API. The version is aligned with
-Commonalities 0.4.0-rc.2 and Identity and Consent Management v0.2.0-rc.2.
+    *  **Connectivity Insights API v0.4.0**
+    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights.yaml&nocors)
+    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights.yaml)
+    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights.yaml)
+
+    *  **Connectivity Insights Subscriptions API v0.4.0**
+    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml&nocors)
+    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)
+    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/connectivity-insights-subscriptions.yaml)
+
+    *  **Application Profiles API 0.3.0**
+    [[View it on ReDoc]](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml&nocors)
+    [[View it on Swagger Editor]](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)
+    [[OpenAPI]](https://raw.githubusercontent.com/camaraproject/ConnectivityInsights/r1.2/code/API_definitions/application-profiles.yaml)
+
+### Added
+* n/a
+### Changed
+* n/a
+
+**Full Changelog**: https://github.com/camaraproject/connectivityInsights/compare/r1.1...r1.2
+
+
+# r1.1
+## Release Notes
+
+This release contains the definition and documentation of
+* Connectivity Insights API v0.4.0
+* Connectivity Insights Subscriptions API v0.4.0
+* Application Profiles API v0.3.0
```

Comment:
The section related to r1.1 should stay unchanged and in-line with https://github.com/camaraproject/ConnectivityInsights/releases/tag/r1.1

```suggestion
* Connectivity Insights API v0.4.0 - rc.1 
* Connectivity Insights Subscriptions API v0.4.0 - rc.1
* Application Profiles API v0.3.0 - rc.1
```

Reply:
issue #99 created to track this.